_content/essays/kyangpen-namkha-dzong_milarepa.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kyangpen Namkha Dzong"
+authors:
+  - "Milarepa"
```

Comment:
Ah yes. You'll either need to use "Jetsun Milarepa" (with a space) or you'll need to make a new _author entry.  I recommend the latter as Milarepa is so important!